nntrainer/optimizers/lr_scheduler_linear.h

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Hyunwoo LEE <dlgusdn0414@snu.ac.kr>
+ *
+ * @file   lr_scheduler_linear.h
+ * @date   11 November 2024
+ * @brief  This is Linear Learning Rate Scheduler class
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Hyunwoo LEE <dlgusdn0414@snu.ac.kr>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#ifndef __LEARNING_RATE_SCHEDULER_LINEAR__
+#define __LEARNING_RATE_SCHEDULER_LINEAR__
+#ifdef __cplusplus
+
+#include <string>
+
+#include <common_properties.h>
+#include <lr_scheduler.h>
+
+namespace nntrainer {
+
+/**
+ * @class   Linear Learning Rate Scheduler class
+ * @brief   class for Linear Learning Rate Schedulers
+ */
+class LinearLearningRateScheduler : public LearningRateScheduler {
+
+public:
+  /**
+   * @brief Construct a new Linear Learning Rate Scheduler object
+   */
+  LinearLearningRateScheduler();
+
+  /**
+   * @copydoc LearningRateScheduler::getLearningRate(size_t iteration) const
+   */
+  virtual double getLearningRate(size_t iteration) override;
```

Comment:
how about remove "virtual" keyword on this function?

as far as I know, many C++ style guides recommend specifying either 'override' or 'final', but not using 'virtual'.

Reply:
Removed 'virtual' keyword